api/api.go

```diff
@@ -60,6 +62,9 @@ func NewRecipeAPI(ctx context.Context, cfg config.Configuration, router *mux.Rou
 	if api.EnableMongoImport {
 		api.Router.HandleFunc("/allrecipes", api.AddAllRecipeHandler).Methods("POST")
 	}
+	if api.EnableAuthImport {
+		api.Router.HandleFunc("/recipes/{id}", api.AddRecipeHandler).Methods("POST")
```

Comment:
It seems unusual to have the ID passed in the URL when 'posting' a new resource. As the ID is already in the request I think it may be cleaner to not have the ID in the URL when posting a new recipe? (may well just be my opinion!)

Reply:
Exactly. That is exactly what I thought too and initially, I wrote the code so that the endpoint doesn't have an ID passed. But when I was looking at the datasetAPI, an ID is passed for the POST request, therefore, I assumed that it is the standard convention. Shall we agree to avoid the ID to be passed in the URL?

Reply:
I think it's preferable to not have it in the URL. I would double check with one of the tech leads though in case this is something that has been decided before.

Reply:
Looked at RESTful documentation and ID does not need to be passed in

Reply:
Done in commit _"Remove the passing of ID in endpoint URL of adding recipes"_